.kokoro/release.sh

```diff
@@ -51,7 +54,28 @@ then
   echo "Pushing NuGet packages"
   # Push the changes to nuget.
   cd ./releasebuild/nuget
-  for pkg in *.nupkg; do dotnet nuget push -s https://api.nuget.org/v3/index.json -k $NUGET_API_KEY $pkg; done
+  for pkg in *.nupkg
+  do
+    # Work out just the package ID based on the filename.
+    pkg_id=$(echo $pkg | sed -r 's/^(.*)\.([0-9]+\.[0-9]+\.[0-9]+(-.*)?)\.nupkg$/\1/g')
+    # Work out the package owner based on apis.json and the package ID
+    default_package_owner=$([[ $pkg == Google.Cloud* ]] && echo google-cloud || echo google-api-packages)
+    package_owner=$PYTHON3 ../tools/getapifield.py ../apis/apis.json $pkg_id --default=$default_package_owner
```

Comment:
Can't you add the finding of the default owner logic to getapifield.py? Seems like a better place that will allow for more reuse if needed.

Reply:
I'd prefer not to put it into getapifield.py - that was intended to be utterly generic. I could create another Python tool to do it if we wanted to - although the part about the NuGet API key case would still be in here.
Perhaps leave it here until we have a second place we need it? I can't think of reasons we need the package owner other than when pushing packages, at the moment.

Reply:
Yep, that's fine.